modules/virt-booting-efi-mode.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * virt-efi-mode-for-vms.adoc
+
+[id="virt-booting-vms-efi-mode_{context}"]
+= Booting virtual machines in EFI mode
+
+Booting a virtual machine in EFI mode allows for a quicker boot process.
+
+[NOTE]
+====
+{VirtProductName} requires System Management Mode (`SMM`) to be enabled for Secure Boot in EFI mode to occur.
+====
+
+.Procedure
+To boot a VM in EFI Mode:
+
+. Create a YAML file that defines a VM object or a Virtual Machine Instance (`VMI`) object. Use the firmware stanza of the example YAML file:
++
+.Booting in `EFI` mode with secure boot active
+[source,yaml]
+----
+  apiversion: kubevirt.io/v1
+  kind: VirtualMachineInstance
+  metadata:
+    labels:
+      special: vmi-secureboot
+    name: vmi-secureboot
+  spec:
+    domain:
+      devices:
+        disks:
+        - disk:
+            bus: virtio
+          name: containerdisk
+      features:
+        acpi: {}
+        smm:
+          enabled: true
+      firmware:
+        bootloader:
+          efi:
+            secureBoot: true
+...
+----
+
+. Load the YAML file in the Command Line Interface (`CLI`) using the corresponding user operation:
++
+[source,terminal]
+----
+$ oc  create -f my-vm-def-file.yaml
```

Comment:
extra space between `oc` and `create`

(I think we usually do `oc apply`. If it must be `create`, I'd change the previous step to "Create the VM or VMI by running the following command:")